acp/acp_local.go

```diff
@@ -0,0 +1,365 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package acp
+
+import (
+	"context"
+
+	protoTypes "github.com/cosmos/gogoproto/types"
+	"github.com/sourcenetwork/immutable"
+	"github.com/sourcenetwork/sourcehub/x/acp/embedded"
+	"github.com/sourcenetwork/sourcehub/x/acp/types"
+	"github.com/valyala/fastjson"
+
+	"github.com/sourcenetwork/defradb/errors"
+	"github.com/sourcenetwork/defradb/logging"
+)
+
+var (
+	_ ACPModule = (*ACPLocalEmbedded)(nil)
+)
+
+// ACPLocalEmbedded implements the ACPModule interface in an embedded manner to allow
```

Comment:
suggestion: It is not clear what you mean by `embedded manner` here, can you make this clearer please? Or remove the phrase if you feel like it is covered by the rest of the sentence

Reply:
- [ ] Remove usage of `Embedded` as it causes confusion from everywhere
- [ ] Change to ACPLocal